h2o-core/src/main/java/water/fvec/ByteVec.java

```diff
@@ -54,22 +54,26 @@
     }
   }
 
-  /** Open a stream view over the underlying data  */
+  /**
+   * Open a stream view over the underlying data
+   */
   public InputStream openStream(final Key job_key) {
-    return new InputStream() {
-      final long [] sz = new long[1];
+    InputStream is = new InputStream() {
+      final long[] sz = new long[1];
```

Comment:
Unrelated comment: why `sz` is here? Looking into code (3.10.0.4), it is never read (?). Also `long[1]` instead of `long`?
Is it memory padding?